controllers/complianceeventsapi/server.go

```diff
@@ -115,6 +120,8 @@ func init() {
 	ErrInvalidQueryArg = fmt.Errorf(
 		"an invalid query argument was provided, choose from: %s", strings.Join(validQueryArgs, ", "),
 	)
+
+	utilruntime.Must(clusterv1.AddToScheme(scheme.Scheme))
```

Comment:
You need to put this in an `init` like in `main.go`, or else if this starts failing, it would cause the pod to crash during the execution instead of at startup.

Reply:
`	utilruntime.Must(clusterv1.AddToScheme(scheme.Scheme))` in init function... should I use main scheme?